crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/cucumber/StatelessSteps.java

```diff
@@ -0,0 +1,175 @@
+package uk.gov.justice.laa.crime.meansassessment.cucumber;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.justice.laa.crime.meansassessment.model.common.stateless.DependantChild;
+import uk.gov.justice.laa.crime.meansassessment.model.common.stateless.StatelessApiResponse;
+import uk.gov.justice.laa.crime.meansassessment.service.stateless.FrequencyAmount;
+import uk.gov.justice.laa.crime.meansassessment.service.stateless.StatelessFullResult;
+import uk.gov.justice.laa.crime.meansassessment.service.stateless.StatelessInitialResult;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.*;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless.AgeRange;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless.StatelessRequestType;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.justice.laa.crime.meansassessment.util.RequestBuilderUtils.buildRequestGivenContent;
+
+public class StatelessSteps {
+    private static final String MEANS_ASSESSMENT_ENDPOINT_URL = "/api/internal/v2/assessment/means";
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private CucumberRequestData requestData;
+
+    @When("^I call the stateless CMA endpoint$")
+    public void call_stateless_cma() throws Exception {
```

Comment:
I think you've been writing too much Ruby! ;)

Reply:
This was generated for me by IDEA, I've no idea why

Reply:
Weird, either way it would be good if we could change this to camel case.